src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebGitRouteConfig.kt

```diff
@@ -0,0 +1,17 @@
+package org.vaccineimpact.orderlyweb.app_start.routing.web
+
+import org.vaccineimpact.orderlyweb.*
+import org.vaccineimpact.orderlyweb.app_start.RouteConfig
+import org.vaccineimpact.orderlyweb.controllers.web.GitController
+
+object WebGitRouteConfig : RouteConfig
+{
+    private val runReports = setOf("*/reports.run")
+    override val endpoints: List<EndpointDefinition> = listOf(
+            WebEndpoint("/git/branch/:branch/commits/",
```

Comment:
Atm route in API is `/git/commits/` with `branch` a param see e.g. https://github.com/vimc/orderly.server/blob/master/tests/testthat/test-z-integration.R#L279

Reply:
That should be OK, the OW and Orderly paths don't need to match. Though this approach was a bit more RESTful but I'll make sure the Orderly server client uses the right path when we start getting the real data.